mailer.rst

```diff
@@ -65,6 +65,10 @@ over SMTP by configuring the DSN in your ``.env`` file (the ``user``,
             ]);
         };
 
+.. caution::
+
+    If your user or password has special characters such as @ or #, use encoding characters such as% 40 or% 23.
+
```

Comment:
Can you add this paragraph to the `notifier.rst` file too? Thanks

Reply:
Not sure what others say, but wouldn't it be enough to mention "if If your user or password has special characters ... based on URL.....the use urlencode" ? instead of mention all characters?

Reply:
@xabbuh wdyt?

Reply:
I like the wording as it is now and adding this to the notifier docs sounds good to me. 👍

Reply:
Thanks